app/javascript/dashboard/routes/dashboard/notifications/components/NotificationPanel.vue

```diff
@@ -201,7 +196,7 @@ export default {
             <fluent-icon
               icon="chevron-left"
               size="16"
-              :class="notificationPanelFooterIconClass"
+              class="rtl:-mr-3 ltr:-ml-3"
```

Comment:
@iamsivin if these `ltr` and `rtl` directives are working fine, let's use it across the app

Reply:
@scmmishra We are using this `ltr` and `rtl` directives almost everywhere. It's done at the time while moved from CSS to Tailwind, Only some cases like [this](https://github.com/chatwoot/chatwoot/pull/9924/files#diff-caa95f34e5ac6da66759094fd4823106f9ee28b67477693a23a85973b7cd54c8R94) we are using it. Do you have any idea?

Reply:
For now it seems fine, it's out of scope for this PR atleast. We can take a look at it later